src/components/app/app.scss

```diff
@@ -1,5 +1,24 @@
+@import "ui/variables";
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap");
+
+html,
 body {
-  font-family: "Helvetica", sans-serif;
+  color: $font-color-1;
+  font-family: "Lato", sans-serif;
+  font-size: 16px;
+  margin: 0;
+  height: 100%;
+}
```

Comment:
this is a question from my own unfamiliarity with CSS:
what happens if another app component defines a `html` or `body` styling? Like, how doe that affect all the other definitions for the tag's style

Reply:
since it's cascading, the most recent definition takes priority unless there was an override. since the app component is the main component that wraps every other component, any other css that tries defining the same styles would override app.scss's